scipy/optimize/_linprog_ip.py

```diff
@@ -708,8 +708,7 @@ def _ip_hsd(A, b, c, c0, alpha0, beta, maxiter, disp, tol, sparse, lstsq,
     if disp:
         _display_iter(rho_p, rho_d, rho_g, "-", rho_mu, obj, header=True)
     if callback is not None:
-        x_o, fun, slack, con, _ = _postsolve(x/tau, postsolve_args,
-                                                tol=tol)
+        x_o, fun, slack, con = _postsolve(x/tau, postsolve_args)
```

Comment:
I see that `tol` was not used at all in `_postsolve`

Reply:
Yes, the `tol`-argument is not used in `_postsolve()`.